usecases/auth/authorization/rbac/manager.go

```diff
@@ -287,7 +323,11 @@ func (m *manager) Snapshot() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func (m *manager) Restore(b []byte) error {
+func (m *Manager) Restore(b []byte) error {
+	// restore is only needed when RBAC was enabled during backup
+	if m == nil || b == nil || len(b) == 0 {
+		return nil
+	}
```

Comment:
whas this the reason for the panic? We already have the len(b)==0 check below (line 337) so we can remove one of them

Reply:
Yes, but I don't see a problem with multiple checks like this.  Especially when someone else can add some more code, or move some around, and then the condition triggers again

Reply:
The other checks is 7 lines below and does exactly the same - do you just want to remove the other check and move the comment from there up?

(Note that len() can handle nil, eg `b == nil || len(b) == 0` does exactly the same as `len(b) == 0`)